src/main/java/com/iota/iri/conf/APIConfig.java

```diff
@@ -9,43 +9,59 @@
 
 
     /**
-     * @return {@value Descriptions#PORT}
+     * Default Value: {@value BaseIotaConfig.Defaults#API_PORT}
```

Comment:
the default value name does not match the description name.
API_PORT vs PORT
(I think I asked to change this one :) )